src/TwigExtension.php

```diff
@@ -142,6 +161,11 @@ public function renderUrlFromPath(string $path = null) : string
      *
      * Usage: {{ asset('path/to/asset/name.ext', version=3) }}
      * Generates: path/to/asset/name.ext?v=3
+     *
+     * @param string      $path
+     * @param string|null $version
+     *
+     * @return string
```

Comment:
Please revert the changes to this docblock.

We only document parameters and return values that:

- do not have typehints, and/or
- have additional narrative descriptions.

None of these fall in those categories.